.gitignore

```diff
@@ -5,3 +5,4 @@ build/
 node_modules/
 package-lock.json
 *.swp
+/.idea
```

Comment:
You can put this in a global `gitignore` instead.

Reply:
I removed this. I often checkin the webstorm files in my own repos which is why I added it here. Not a problem.